backend/api/Areas/Admin/Mapping/User/UserMap.cs

```diff
@@ -45,8 +45,8 @@ public void Register(TypeAdapterConfig config)
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Style", "IDE0060:Remove unused parameter", Justification = "Required for signature")]
         private void UpdateUser(Model.UserModel model, Entity.User entity)
         {
-            entity.Agencies.ForEach(a => a.UserId = entity.Id);
-            entity.Roles.ForEach(r => r.UserId = entity.Id);
+            entity.Agencies.Where(a => a != null).ForEach(a => a.UserId = entity.Id);
```

Comment:
For some reason I was running into null exceptions.  But if I debugged I never ran into them.  There appears to be some inconsistency in the mapper implementation, but this resolved it for our requirements.